admin/views/tool-file-editor.php

```diff
@@ -149,21 +152,23 @@
 			'robots.txt'
 		);
 		echo '</em></p>';
-		echo '<textarea class="large-text code" disabled="disabled" rows="15" name="robotsnew">', esc_textarea( $content ), '</textarea><br/>';
+		echo '<textarea class="yoast-field-group__textarea" disabled="disabled" rows="15" name="robotsnew">', esc_textarea( $content ), '</textarea><br/>';
```

Comment:
Removing the `code` CSS class will change the type face used. In WordPress, `code` when an input field/textarea contains code. It will use a monospace type face `font-family: Consolas, Monaco, monospace;` which is more appropriate for code readability. Has this been considered?

Side by side quick comparison without vs. with `code`:

<img width="1024" alt="Screenshot 2020-04-28 at 14 38 23" src="https://user-images.githubusercontent.com/1682452/80488481-b122cb00-895e-11ea-9cf8-76675a284c23.png">